src/banded/BandedMatrix.jl

```diff
@@ -1007,3 +1007,59 @@ function resize(A::BandedSubBandedMatrix, n::Integer, m::Integer)
     l,u = bandwidths(A)
     _BandedMatrix(reshape(resize!(vec(copy(bandeddata(A))), (l+u+1)*m), l+u+1, m), n, l,u)
 end
+
+function sum(A::BandedMatrix; dims=:)
+    if(dims == :)
```

Comment:
This is not standard style. A more standard notation would be:
```suggestion
    if dims == :
```